priv/repo/migrations/20210226085650_change_assessment_id_to_question_id_in_submission_votes_table.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Cadet.Repo.Migrations.ChangeAssessmentIdToQuestionIdInSubmissionVotesTable do
+  use Ecto.Migration
+
+  def change do
+    alter table("submission_votes") do
+      add_if_not_exists(:question_id, references(:questions))
+      remove_if_exists(:assessment_id, references(:assessments))
+    end
+  end
+end
```

Comment:
Please merge these changes into the initial migration

(Yes, migrations shouldn't be changed once they have been deployed. But they have not in this case, so in the interest of keeping this clean, please do so)